engine/src/lib.rs

```diff
@@ -177,6 +178,23 @@ mod contract {
         io.return_output(&state::get_state(&io).sdk_unwrap().chain_id);
     }
 
+    #[no_mangle]
+    pub extern "C" fn get_upgrade_delay_blocks() {
+        let mut io = Runtime;
+        let state = state::get_state(&io).sdk_unwrap();
+        io.return_output(&state.upgrade_delay_blocks.to_le_bytes());
+    }
+
+    #[no_mangle]
+    pub extern "C" fn set_upgrade_delay_blocks() {
```

Comment:
When you add a new mutable method to the contract you also need to make sure the standalone engine knows how to process it by adding it to the `TransactionKind` enum. See the [set_owner PR for example](https://github.com/aurora-is-near/aurora-engine/pull/690).